mmv1/products/compute/UrlMap.yaml

```diff
@@ -1945,6 +1948,42 @@ properties:
           required: true
           description: The backend service or backend bucket link that should be matched by this test.
           custom_expand: 'templates/terraform/custom_expand/reference_to_backend.erb'
+        - !ruby/object:Api::Type::Array
+          name: 'headers'
+          description: |
+            HTTP headers for this request. If headers contains a host header, then host must also match the header value.
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'name'
+                required: true
+                description: |
+                  Header name.
+              - !ruby/object:Api::Type::String
+                name: 'value'
+                required: true
+                description: |
+                  Header value.
+        - !ruby/object:Api::Type::String
+          name: 'expected_output_url'
+          required: true
```

Comment:
Based on my read of these fields, this field should not be required (which would also be a breaking change).